Project-Aurora/Project-Aurora/Devices/Uniwill/UniwillDevice.cs

```diff
@@ -0,0 +1,284 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Aurora.Devices.RGBNet;
+using Aurora.Settings;
+using Microsoft.Win32;
+using UniwillSDKDLL;
+
+namespace Aurora.Devices.Uniwill
+{
+     
+    enum GAMECENTERTYPE
+    {
+        NONE = 0,
+        GAMINGTCENTER = 1,
+        CONTROLCENTER = 2
+        
+    }
+
+
+    public class UniwillDevice : Device
+    {
+        
+       // Generic Variables
+        private string devicename = "Uniwill";
+        private bool isInitialized = false;
+
+        private System.Diagnostics.Stopwatch watch = new System.Diagnostics.Stopwatch();
+        private long lastUpdateTime = 0;
+
+        System.Timers.Timer regTimer;
+        const string Root = "HKEY_LOCAL_MACHINE";
+        const string subkey = @"SOFTWARE\OEM\Aurora";
+        const string keyName = Root + "\\" + subkey;
+        int SwitchOn = 0;
+        List<AuroraInterface> DeviceList = new List<AuroraInterface>();
```

Comment:
this isn't being used, what is it for?

Reply:
Actually, We have other devices, but it’s just not ready right now.